DetectorDescription/DDCMS/test/DDFilteredView.cppunit.cc

```diff
@@ -94,7 +94,8 @@ void testDDFilteredView::checkFilteredView() {
   cout << "Get attl from hf as double values:\n";
   vector<double> attl = fview.get<vector<double>>("hf", "attl");
   int count(0);
-  for (auto const& i : attl) {
+  for (double i : attl) {
```

Comment:
@cvuosalo - `std::transform` seems to be more natural way of doing it.

Reply:
The current method is very simple. It looks to me that `transform` would require declaring an extra vector and a function -- not really worth it when the loop iterates over only two elements.